syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -34,7 +34,7 @@
 			"patterns": [
 				{
 					"name": "keyword.control.scl",
-					"match": "!|\\$|\\&|<\\=|>\\=|<>|<|>|(\\*\\*)|(\\=\\=)|!|\\&\\&|\\*|\\/|\\-|\\+|:|\\=|\\#|\\b(void)\\b"
+					"match": "!|\\$|\\&|<\\=|>\\=|<>|<|>|(\\*\\*)|(\\=\\=)|!|\\&\\&|\\*|\\/|\\-|\\+|:|\\=|\\b(void)\\b"
```

Comment:
This was added in #12, and I don't see the reason for removing it. Please rebase this file

Reply:
I agree that this is useless now that you have implemented recognition of #variable.